hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -202,11 +210,11 @@ private OzoneManagerDoubleBuffer(OMMetadataManager omMetadataManager,
         OzoneManagerDoubleBufferMetrics.create();
     this.indexToTerm = indexToTerm;
     this.flushNotifier = flushNotifier;
-
+    this.threadPrefix = threadPrefix;
     isRunning.set(true);
     // Daemon thread which runs in background and flushes transactions to DB.
     daemon = new Daemon(this::flushTransactions);
-    daemon.setName("OMDoubleBufferFlushThread");
+    daemon.setName(threadPrefix + "OMDoubleBufferFlushThread");
```

Comment:
similar changes for OzoneDelegationTokenSecretManager is missing for daemon being created

Reply:
Done !